lldb/include/lldb/API/SBValue.h

```diff
@@ -193,7 +188,7 @@ class LLDB_API SBValue {
   ///     and also if the target can be run to figure out the dynamic
   ///     type of the child value.
   ///
-  /// \param[in] can_create_synthetic
+  /// \param[in] use_synthetic
```

Comment:
I think this change makes sense, but there must be a better term for this.  use_synthetic doesn't actually tell you anything about what you are going to use synthetic FOR, and is confusing because it does NOT mean "use the children produced by the synthetic child provider for this type", which is almost always what we mean by "using synthetic".
Seems to me this bool is forcing "treat_as_array" behavior for a pointer.  Would that be an accurate name for what this parameter controls?

Reply:
Sounds good. I wasn't too happy with the name myself.